css/css-color/t421-rgb-hex3-expand-b-ref.html

```diff
@@ -17,8 +17,8 @@
   <p>The left and right cells in each row of the following table should be slightly <em>different</em> colors.  The right side should be slightly darker than the left.</p>
 
   <table>
-    <tr><td style="background: #e09020">&nbsp;</td><td style="background: #e09020">&nbsp;</td></tr>
-    <tr><td style="background: #f0b000">&nbsp;</td><td style="background: #f0b000">&nbsp;</td></tr>
-    <tr><td style="background: #308010">&nbsp;</td><td style="background: #308010">&nbsp;</td></tr>
+    <tr><td style="background: #ee9922">&nbsp;</td><td style="background: #e09020">&nbsp;</td></tr>
```

Comment:
Those three changes are clearly correct, astonished the earlier values were in the test to be honest